readme.md

```diff
@@ -33,6 +36,14 @@ The destination folder is optional, and defaults to the current folder name. Wat
     apis      programs
     github readme.md
 
+### Adding Authentication
+To use authenticated requests you must first [create a github](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) api token on your github account. You do not need to provide any api scopes for the token unless you plan on accessing private repositories.
+
+    github auth <user> [<api token> | -d]
+
+The delete argument is optional and will delete specified user when defined.  The user and token arguments maybe undefined when using the delete option.
```

Comment:
I'd argue that all of these words are fully described by the `[ a | b ]` notation which means "a, b, or nothing at all"